<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiControlUnit</class>
 <widget class="QMainWindow" name="GuiControlUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Duellanlage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_start">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_automatic">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Automatik</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_delete">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LÃ¶schen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_glue">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>500</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kleben</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_connection">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_shoot_time">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>000.0</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_preparation_time">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_runs">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
